config/config.go

```diff
@@ -175,6 +177,19 @@ func loadReceivers(v *viper.Viper, factories map[string]receiver.Factory) (confi
 	// Get the map of "receivers" sub-keys.
 	keyMap := v.GetStringMap(receiversKeyName)
 
+	// Currently there is no default receiver enabled. The configuration must specify at least one receiver to enable
```

Comment:
We have separate validation code. Would it make sense to move this code there?

Also, I would suggest to split this PR into 2 parts: code change and documentation change. It is difficult to review both at the same time.

Reply:
I forgot to add a todo to the validateReceivers that we need to revisit the approach of deleting the disabled receivers because it sounds like there will be a cli way to enable/disable a receiver. 

One reason why the validateReceivers need to happen later on in the process of loading a configuration is loading pipelines checks if a receiver is enabled or not before adding it to its configuration so we can't remove the disabled receivers in this part.

Reply:
Re splitting this pr into 2 -are you okay with doing it for future prs to keep the history of the conversations?

Reply:
I highly recommend to split the PR. You do not need to delete this one, just remove changes that are not relevant to documenting existing behavior.

One PR and one commit per concern is much nicer. This PR is performing 2 mostly independent changes, I do not see good reasons to group them. I'd just remove config validation code changes from this PR and keep everything about documentation and tests which are related to documentation.

Let this be a separate PR:

>Add validation/test to have at least one enabled receiver in configuration.

Reply:
Synced offline - This will be a one off and leave all of the changes together since i ended up fixing code functionality that was related to fixing the docs (ex: ensuring that there is one enabled receiver was mentioned but not enforced in code with no test case either)